apps/ledger-live-mobile/src/dynamicContent/utils.tsx

```diff
@@ -141,3 +142,45 @@ export const mapAsHorizontalContentCard = (card: BrazeContentCard): HorizontalCo
   viewed: card.viewed,
   order: parseInt(card.extras.order) ? parseInt(card.extras.order) : undefined,
 });
+
+export const mapAsSmallSquareContentCard = (card: BrazeContentCard): VerticalContentCard => ({
+  id: card.id,
+  tag: card.extras.tag,
+  title: card.extras.title,
+  description: card.extras.description,
+  image: card.extras.image,
+  price: card.extras.subtitle,
+  size: "S",
+  link: card.extras.link,
+  createdAt: card.created,
+  viewed: card.viewed,
+  order: parseInt(card.extras.order) ? parseInt(card.extras.order) : undefined,
+});
+
+export const mapAsMediumSquareContentCard = (card: BrazeContentCard): VerticalContentCard => ({
+  id: card.id,
+  tag: card.extras.tag,
+  title: card.extras.title,
+  description: card.extras.description,
+  image: card.extras.image,
+  price: card.extras.subtitle,
+  size: "M",
+  link: card.extras.link,
+  createdAt: card.created,
+  viewed: card.viewed,
+  order: parseInt(card.extras.order) ? parseInt(card.extras.order) : undefined,
+});
+
+export const mapAsBigSquareContentCard = (card: BrazeContentCard): VerticalContentCard => ({
+  id: card.id,
+  tag: card.extras.tag,
+  title: card.extras.title,
+  description: card.extras.description,
+  image: card.extras.image,
+  price: card.extras.subtitle,
+  size: "L",
```

Comment:
You can't just give size as parameter ? as only size change ?

Reply:
We also have a mapping for the price but I agree we could have something reused between the mapping functions